lib/service/map_direction.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:convert';
+
+import 'package:concordi_around/model/direction.dart';
+import 'package:concordi_around/model/list_item.dart';
+import 'package:dio/dio.dart';
+import 'package:uuid/uuid.dart';
+
+import '../credential.dart';
+
+class MapDirection {
+
+  var uuid = Uuid();
+  String _sessionToken;
```

Comment:
The concept of sessionTokens do not apply in google web service direction API requests. This should be removed.